test/test_web.py

```diff
@@ -286,7 +288,9 @@ def test_buttons(self) -> None:
         ExampleReadable(int, 42).connect(b4)
 
         page = self.server.page('index')
-        page.add_item(shc.web.widgets.ButtonGroup("My button group", [b1, b2, b3, b4]))
+        page.add_item(
+            shc.web.widgets.ButtonGroup("My button group", cast(Iterable[AbstractButton], [b1, b2, b3, b4]))
+        )
```

Comment:
Same mypy issue here, not sure how to avoid the cast.

Reply:
Same here: I would either declare the list with an explicit type hint or use a `# type: ignore` comment with an explanatory comment.

Reply:
Well, after thinking about it (and having to deal with the other MyPy warnings): The `cast()` is probably better than the type: ignore comment, as it is more specific. And MyPy can also inform me when it is not necessary anymore (using the `--warn-redundant-casts` argument). So I think, your current solution is already good. :)

Reply:
Whoops read this one too late. So I will leave the cast in the `pulseaudio_sink.py` as well, ok?